### PR TITLE
update couch logging format

### PR DESCRIPTION
**Pull request by snopoke**
cross: https://github.com/dimagi/couchdbkit/pull/21
@esoergel

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -939,7 +939,7 @@
             'format': '%(asctime)s %(levelname)s %(module)s %(message)s'
         },
         'couch-request-formatter': {
-            'format': '%(asctime)s [%(username)s:%(domain)s] %(hq_url)s %(method)s %(error_status)s %(path)s %(duration)s'
+            'format': '%(asctime)s [%(username)s:%(domain)s] %(hq_url)s %(method)s %(status_code)s %(content_length)s %(path)s %(duration)s'
```
Comment (dimagimon) [inline settings.py]: E501 line too long (140 > 115 characters)

**Comment by emord** [comment]
I think you need to publish couchdbkit to pypi first

**Comment by esoergel** [comment]
I don't see couchdbkit on pypi yet (guessing it doesn't happen automatically), but after that it should be g2g.

**Comment by snopoke** [comment]
Yes. Need to publish. Hence the 'don't merge'.

**Comment by snopoke** [comment]
:white_check_mark:

**Comment by emord** [comment]
still has don't merge label

**Comment by snopoke** [comment]
removed label

**Pull request merged**